### DISCUSSION TITLE
Returning a result set?

### DISCUSSION THREAD

**Discussion by Jonnokc** [aws/amazon-redshift-python-driver#51] [Q&A] [answered]
has anyone been successful with using this connector to return a result set from a procedure? Not sure if it's currently possible but this would be awesome.

https://docs.aws.amazon.com/redshift/latest/dg/stored-procedure-result-set.html

**Comment by Brooke-white** [comment, marked as answer]
Hi @Jonnokc ,

As of v2.0.886 we've added support for `Cursor.callproc()`. An example of how to use this method can be found in our [test suite](https://github.com/aws/amazon-redshift-python-driver/blob/3638a204b2c16fdcba59a7aaafda5f858ccce56c/test/integration/test_dbapi20.py#L139)
